setup.cfg

```
@@ -20,7 +20,7 @@ package_dir =
 packages = find:
 python_requires = >=3.11
 install_requires =
-    app_server~=0.9.9
+    app_server==0.9.9
```

Please stop using fixed version numbers where there is no need to.
You would have to create a new cli version for each patch in one of these dependencies. But nobody wants that. At least the patch level should IMO always be variable, because here definitely nothing may change in behavior (at least this is the case with packages outside the ViUR world, that work strictly according to semver …).
Minor releases shouldn't actually break anything big. But I would make this dependent on how intensively or deeply you use the module. For example, we only make a few `GET`s with `requests`: this call will still work in the next minor versions, as it is an essential basic method. With `click`, it could be a bit more complicated.

```suggestion
    app_server~=0.9
    click~=8.1.7
    pipfile-requirements~=0.3.0
    requests~=2.31
    semver~=3.0
```